cmd/gin-repod/main_test.go

```diff
@@ -89,7 +89,7 @@ func RunRequest(method string, url string, body io.Reader,
 		return nil, err
 	}
 
-	if header != nil && len(header) > 0 {
+	if len(header) > 0 {
```

Comment:
I kind of like the nil check. Gives me the warm feeling that everything is ok. But the magics of the go seem to support the removal :-)